client/src/app/shared/video/video.service.ts

```diff
@@ -115,6 +117,49 @@ export class VideoService {
       .catch((res) => this.restExtractor.handleError(res))
   }
 
+  baseFeed () {
+    let feed = {}
+
+    for (let item in FeedFormat) {
+      feed[FeedFormat[item]] = VideoService.BASE_FEEDS_URL + item.toLowerCase()
+    }
+
+    return feed
+  }
+
+  getFeed (
+    filter?: VideoFilter
+  ) {
+    let params = new HttpParams()
+    params = this.restService.addRestGetParams(params)
+    let feed = this.baseFeed()
```

Comment:
`const`

Reply:
I don't see how you envision this. both declared variable are meant to be modified in the process.